deploy/stripe-ct/action.yml

```diff
@@ -0,0 +1,83 @@
+name: Deploy Storefront
```

Comment:
Please add information about this action in README.md and more detailed description for our SEs here: https://alokai.atlassian.net/wiki/spaces/UST/pages/288227346/Deployment

Reply:
Done